docs/configuring.rst

```diff
@@ -1,108 +1,210 @@
-* ``prev_next_buttons_location`` String. can take the value ``bottom``, ``top``, ``both`` , or ``None``
-  and will display the "Next" and "Previous" buttons accordingly.
-* ``style_external_links`` Bool. Add an icon next to external links. Defaults to ``False``.
-* ``vcs_pageview_mode`` String. Changes how to view files when using `display_github`, `display_gitlab`, etc.
-  When using Github or Gitlab this can be: `blob` (default), `edit`, or `raw`,
-  on Bitbucket, this can be either: `view` (default) or `edit`.
+    :type: boolean
+    :default: ``True``
 
-TOC Options
-~~~~~~~~~~~
+    Specifies if the navigation includes TOC trees marked with
+    the ``:hidden:`` option
 
-These effect how we display the Table of Contents in the side bar. You can read more about them here: http://www.sphinx-doc.org/en/stable/templating.html#toctree
+.. confval:: titles_only
 
-* ``collapse_navigation`` Bool. With this enabled, you will lose the ``[+]`` drop downs next to each section in the sidebar.
-* ``sticky_navigation`` Bool. This causes the sidebar to scroll with the main page content as you scroll the page.
-* ``navigation_depth`` Int. Indicate the max depth of the tree; by default, 4 levels are included;
-  set it to -1 to allow unlimited depth.
-* ``includehidden`` Bool. Specifies if the sidebar includes toctrees marked with the ``:hidden:`` option
-* ``titles_only`` Bool. If True, removes headers within a page from the sidebar.
+    :type: boolean
+    :default: False
+
+    When enabled, page subheadings are not included in the
+    navigation.
 
 .. note::
-   
-   Setting ``collapse_navigation`` to False and using a high ``navigation_depth``
-   can cause projects with many files and a deep file structure to generate HTML files
-   that are significantly larger in file size and much longer compilation times.
+    Setting :confval:`collapse_navigation` to ``False`` and using a high value
+    for :confval:`navigation_depth` on projects with many files and a deep file
+    structure can cause long compilation times and can result in HTML files that
+    are significantly larger in file size.
+
+.. _table of contents configuration options: http://www.sphinx-doc.org/en/stable/templating.html#toctree
+
+
+..
+    TODO
+    .
+    HTML context options
+    ~~~~~~~~~~~~~~~~~~~~
+
+
+Miscellaneous options
+---------------------
+
+.. confval:: analytics_id
+
+    :type: string
+
+    Change the Google Analytics ID that is included on pages.
+
+.. confval:: canonical_url
+
+    :type: URL
+
+    This will specify a `canonical URL`_ meta link element to tell search
+    engines which URL should be ranked as the primary URL for your
+    documentation. This is important if you have multiple URLs that your
+    documentation is available through. The URL points to the root path of the
+    documentation and requires a trailing slash.
+
+.. confval:: display_version
```

Comment:
Add default here

Reply:
Which is `True` https://github.com/rtfd/sphinx_rtd_theme/blob/master/sphinx_rtd_theme/theme.conf#L15